jena-fuseki2/apache-jena-fuseki/fuseki-server

```diff
@@ -80,7 +80,8 @@ fi
 
 JVM_ARGS=${JVM_ARGS:--Xmx1200M}
 
-exec $JAVA $JVM_ARGS -jar "$JAR" "$@"
+exec $JAVA $JVM_ARGS -cp "$JAR:$FUSEKI_BASE/extra/*" org.apache.jena.fuseki.cmd.FusekiCmd "$@"
```

Comment:
Better to have some testing here - does extra exist, is it a directory, is it accessible (and in Docker the last seem quite important).